contracts/modules/andromeda-address-list/src/contract.rs

```diff
@@ -29,13 +29,17 @@ pub fn instantiate(
     if let Some(actor_permission) = msg.actor_permission {
         let verified_address: Addr = deps.api.addr_validate(actor_permission.actor.as_str())?;
         // Permissions of type "Contract" aren't allowed in the address list contract
```

Comment:
Is this typo? According to the code, Limited permission is not supported and according to the comment, Permissions of type Contract is not supported.

Reply:
Both aren't supported. I will clarify the comment.

Reply:
Hmm. I can see that `actor_permission` is defined as `ActorPermission` which is only accepting `LocalPermission`. I think it is not really necessary to emphasis `Contract` type is not supported.